app/components/add-layer/component.js

```diff
@@ -176,14 +178,14 @@ export default Ember.Component.extend({
             return false;
         },
         drag(el, event) {
+            event.dataTransfer.setData('text/html', event.target.dataset.title) // for firefox 
```

Comment:
This looks weird, can you add more of an explanation in the comment about why FF needs this?

Reply:
it is weird, Firefox has a requirement on their docs about it
https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/setData